<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetscreenact</class>
 <widget class="QWidget" name="widgetscreenact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1884</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="Screenact" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1881</width>
     <height>741</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>0</y>
      <width>641</width>
      <height>741</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="MpColorPlot" name="mfPlot" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>641</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="MpColorPlot" name="mfaPlot" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>641</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="MpColorPlot" name="sftPlot" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>570</y>
       <width>641</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="MpColorPlot" name="wavPlot" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>641</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="echoactTxt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>631</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="curvePlot" name="wavPlotcurve" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>631</width>
       <height>101</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="curvePlot" name="mfPlotcurve" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>631</width>
       <height>91</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1241</width>
      <height>741</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="MpColorPlot" name="actPlot" native="true">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>0</y>
       <width>1261</width>
       <height>741</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MpColorPlot</class>
   <extends>QWidget</extends>
   <header location="global">mpcolorplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>curvePlot</class>
   <extends>QWidget</extends>
   <header location="global">curveplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
